cli/cmd/multicluster.go

```diff
@@ -707,7 +707,7 @@ func newExportServiceCommand() *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:   "export-service",
-		Short: "Exposes a remote service to be mirrored",
+		Short: "Exposes a service to be mirrored",
```

Comment:
I think saying less is better in this description, but I could change it to `Exposes a target service to be mirrored`.